project2/style.css

```diff
@@ -0,0 +1,66 @@
+.title {
+    font-family: 'ZCOOL QingKe HuangYou', cursive;
+    color: white;
+    font-size: 19pt;
+    line-height: 50px;
+    margin-left: 8px;
+    padding-right: 20px;
+
+}
+
+.rate {
+    font-family: 'Khand', sans-serif;
+    font-size: 15pt;
+    display: table;
+    color: white;
+    position: absolute;
```

Comment:
Note that all your `position` rules have a value of `absolute` -- they need a container or wrapping element around them to be relative to, otherwise they are positioned relative to the browser window/page, which only works when one card is on the page -- if you added another movie card to this page, it would overlap the Spiderman perfectly because they're both positioned relative to the page.

See [this CodePen](https://codepen.io/angeliquejw/pen/193ea4d0e758aa0c5cb337865543c821) for an example of how to simplify your markup and position things relative to a wrapper `div`. I made a lot of comments/notes in the markup and CSS, too, to hopefully better explain this.